### ISSUE TITLE
An unexpected keyword argument 'default_range'

### ISSUE THREAD

**Issue by deepakunni3** [biolink/biolink-model#226]
It looks like the `metamodel/utils/loadschema.py` doesn't recognize `default_range` field in `biolink-model.yaml`.

@hsolbrig Any ideas?

```
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.7/bin/kgx", line 11, in <module>
    load_entry_point('Knowledge-Graph-Exchange==0.0.1', 'console_scripts', 'kgx')()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 489, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2793, in load_entry_point
    return ep.load()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2411, in load
    return self.resolve()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2417, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "/Library/Frameworks/Python.framework/Versions/3.7/bin/translator_kgx.py", line 4, in <module>
    __import__('pkg_resources').run_script('Knowledge-Graph-Exchange==0.0.1', 'translator_kgx.py')
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 666, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 1446, in run_script
    exec(code, namespace, namespace)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/Knowledge_Graph_Exchange-0.0.1-py3.7.egg/EGG-INFO/scripts/translator_kgx.py", line 1, in <module>
    import kgx
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/Knowledge_Graph_Exchange-0.0.1-py3.7.egg/kgx/__init__.py", line 7, in <module>
    from .rdf_transformer import RdfTransformer, ObanRdfTransformer, RdfOwlTransformer
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/Knowledge_Graph_Exchange-0.0.1-py3.7.egg/kgx/rdf_transformer.py", line 11, in <module>
    from kgx.rdf_graph_mixin import RdfGraphMixin
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/Knowledge_Graph_Exchange-0.0.1-py3.7.egg/kgx/rdf_graph_mixin.py", line 6, in <module>
    from kgx.utils.rdf_utils import find_category, category_mapping, equals_predicates, property_mapping, predicate_mapping, process_iri, make_curie, is_property_multivalued
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/Knowledge_Graph_Exchange-0.0.1-py3.7.egg/kgx/utils/rdf_utils.py", line 9, in <module>
    tk = Toolkit()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bmt-0.0.7-py3.7.egg/bmt/__init__.py", line 38, in __init__
    self.generator = ToolkitGenerator(schema)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/bmt-0.0.7-py3.7.egg/bmt/toolkit_generator.py", line 16, in __init__
    super().__init__(schema, None)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/BiolinkMG-0.3.0-py3.7.egg/metamodel/utils/generator.py", line 29, in __init__
    self.schema = schema if isinstance(schema, SchemaDefinition) else SchemaLoader(schema).resolve()
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/BiolinkMG-0.3.0-py3.7.egg/metamodel/utils/schemaloader.py", line 13, in __init__
    self.schema = load_raw_schema(data, base_dir=base_dir)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/BiolinkMG-0.3.0-py3.7.egg/metamodel/utils/loadschema.py", line 36, in load_raw_schema
    return load_raw_schema(response)
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/BiolinkMG-0.3.0-py3.7.egg/metamodel/utils/loadschema.py", line 51, in load_raw_schema
    for sname, sdef in {k: SchemaDefinition(name=k, **v) for k, v in schemadefs.items()}.items():
  File "/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/BiolinkMG-0.3.0-py3.7.egg/metamodel/utils/loadschema.py", line 51, in <dictcomp>
    for sname, sdef in {k: SchemaDefinition(name=k, **v) for k, v in schemadefs.items()}.items():
TypeError: __init__() got an unexpected keyword argument 'default_range'
```

**Comment by hsolbrig** [comment]
You need to pull the latest version of https://github.com/NCATS-Tangerine/biolink-model-toolkit (bmt) Question:  was pypi updated after the latest merge?

**Comment by deepakunni3** [comment]
Fixed via new 0.1.0 release of bmt: https://pypi.org/project/bmt/0.1.0/
